app/views/schedules/new.html.erb

```diff
@@ -4,8 +4,8 @@
 <div class="row">
   <div class="col-md-8 col-md-offset-2">
     <%= form_for(@schedule) do |f| %>
-        <%= render partial: 'fields', locals: {f: f, object: @schedule} %>
-        <%= button_tag 'Add', type: :submit, class: 'btn btn-sm btn-primary' %>
+        <%= render partial: 'fields', locals: {f: f, object: @schedule } %>
+        <%= button_tag 'Add Root Task', type: :submit, class: 'btn btn-sm btn-primary' %>
```

Comment:
Since the user is also adding the Name, I don't think you need to change this. The button is referencing that the user is really adding the _Schedule_, not the components of the Schedule.